docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1255 @@
+        job will need enough memory to run. For instance, if
+        <emphasis>maxActive</emphasis> is set to 2000, you could submit a very
+        big job and in that case spawn some 2000 jobs using a considerable
+        amount of resources, which could potentially run up a rather expensive
+        compilation bill, again depending on your cloud provider and your
+        billing plan.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Sasha">
+        <title>Sasha</title>
+
+        <para>Sasha is an outlier as it is a dictionary type structure and not
+        a list. You can't have more than one archiver or dfuwu-archiver as
+        that is a value limitation, you can choose to either have the service
+        or not (set the 'disabled' value to true). The Sasha component is an
+        exception to the norm for HPCC Systems components.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Thor">
+        <title>Thor</title>
+
+        <para>Thor instances run dynamically. As do the other components in
+        containers. In Thor's case, a collection of pods is spawned (manager +
+        N workers) when jobs are queued to it. When idle there are no worker
+        (or manager) pods running.</para>
+
+        <para>In the case of Thor (components) there are similarities with the
+        ECLCC Server. Thor is also a list, and if that list has a value
+        of</para>
+
+        <programlisting>-name: thor </programlisting>
+
+        <para>Or</para>
+
+        <programlisting>-name: thor_50</programlisting>
+
+        <para>If you have a small job you don't need 400 nodes when 10 would
+        do. If you wanted 50 nodes set the number of workers the <emphasis
+        role="bold">numWorkers</emphasis> value to 50 and you would have 50
+        nodes. In a case when multiple Thors would be useful:</para>
+
+        <para><programlisting>thor:
+- name: thor
+  prefix: thor
+  numWorkers: 2</programlisting></para>
+
+        <para>Continuing on this example, if you wanted a 50-way Thor, you set
+        <emphasis>numWorkers:</emphasis> value to 50 and that would deploy a
+        50 node Thor. In doing so, ideally you should rename the resource to
+        something which clearly describes it, such as
+        <emphasis>thor_50</emphasis> as in the above example. Updating the
+        <emphasis>numWorkers</emphasis> value will restart the Thor agent
+        listening to the queue, causing all new jobs to use the new
+        configuration.</para>
+
+        <para>Similarly to the <emphasis>maxActive</emphasis> values in the
+        ECLCC Server, Thor has some analogous options.</para>
+
+        <para><emphasis role="bold">MaxJobs</emphasis>: controls the number of
+        jobs, specifically <emphasis>MaxJobs</emphasis> sets maximum number of
+        jobs.</para>
+
+        <para><emphasis role="bold">MaxGraphs</emphasis>: likewise
+        <emphasis>MaxGraphs</emphasis> also limits the maximum amount of
```

Comment:
This doesn't help me understand what the difference between the settings is (I don't know, but I can guess). I think this needs to explain that thor queries may only spend some of their time running graphs.  I'm not sure if having two different settings is very useful.  @jakesmith ?

Reply:
The idea was that jobs may spend significant amount of time outside of graphs, e.g. waiting on workflow state (outside of the Thor engine itself), like on persist points, superfile ops. etc., so it makes sense for Thor's to have a higher limit of concurrent jobs (maxJobs) than graphs (maxGraphs / Thor instances).

I agree, it is a bit confusing, but I also think we want this disctinction, because Thor instances (graphs) are expensive (lots of pods/higher resources), and workflow pods (jobs) are relatively cheap.

I wonder if we should change it, so maxActive means what maxGraphs (Thor instances) means now,
but there is also still a maxJobs, which is not set by default and is automatically set to e.g. 50% more than maxActive.

@ghalliday - thoughts?

Reply:
I think it is ok if is explained clearly.

"since not all jobs submit graphs - but some jobs may be contained in ECL Agent and may not want Thor graphs."

This isn't really correct - Jake's comment above is better.  The issue is that the thor jobs are not executing graphs all the time, but they will some of the time.  Another example is spraying files.